bundler/lib/dependabot/bundler/file_fetcher.rb

```diff
@@ -55,21 +58,33 @@ def uniq_files(fetched_files)
       end
 
       def check_required_files_present
-        return if gemfile || gemspecs.any?
+        return if gemfiles.any? || gemspecs.any?
 
         path = Pathname.new(File.join(directory, "Gemfile")).
                cleanpath.to_path
         raise Dependabot::DependencyFileNotFound, path
       end
 
-      def gemfile
-        @gemfile ||= fetch_file_if_present("gems.rb") ||
-                     fetch_file_if_present("Gemfile")
```

Comment:
It looks like we used to support either Bundler 2.0 `gems` files OR old-style `Gemfile`s. What do you think about returning either the `gems.rb`/`gems.locked` files if these exist or else continuing with the new `Gemfile` regex you've added?